### ISSUE TITLE
[Feature] Add a Readme

### ISSUE THREAD

**Issue by ChasarooniZ** [TimWeidner/everybody-liked-this#7]
A Readme would help with knowing what this module is without having to find it on the addons page

**Comment by DeviousHearts** [comment]
Yes, knowing what the purpose of the module is and how it can specifically help gamemasters is crucial knowledge that can help us evaluate whether or not we need to add it to a game or not.
